src/app/core/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,60 @@
+import { forwardRef, Ref, type ReactNode } from "react";
```

Comment:
@subject026 FYI new component for rendering Badges they look like this 

<img width="332" alt="Screenshot 2024-11-13 at 3 27 00 PM" src="https://github.com/user-attachments/assets/9405047e-8dce-4bb0-8ac9-bc24f389c9cc">